apps/ios/Shared/Model/ChatModel.swift

```diff
@@ -326,6 +328,16 @@ final class ChatModel: ObservableObject {
         chats.insert(chat, at: position)
     }
 
+    func dismissConnReqView(_ id: String) {
+        if let connReqInv = connReqInv,
+           let c = getChat(id),
+           case let .contactConnection(contactConnection) = c.chatInfo {
+            if connReqInv == contactConnection.connReqInv {
+                dismissAllSheets()
```

Comment:
does it have to be in separate if?
```suggestion
           case let .contactConnection(contactConnection) = c.chatInfo,
           connReqInv == contactConnection.connReqInv {
            dismissAllSheets()
```